bitnami/fluent-bit/Chart.yaml

```
@@ -29,4 +29,4 @@ maintainers:
 name: fluent-bit
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/fluent-bit
-version: 3.0.0
+version: 3.0.1
```

Please bump minor version instead:
```suggestion
version: 3.1.0
```